src/core/thread/fiber.d

```diff
@@ -1042,23 +1041,23 @@ private:
         {
             VirtualFree( m_pmem, 0, MEM_RELEASE );
         }
-        else
+        else version (Posix)
         {
             import core.sys.posix.sys.mman; // munmap
+            import core.stdc.stdlib : free;
 
             static if ( __traits( compiles, mmap ) )
             {
                 munmap( m_pmem, m_size );
             }
-            else static if ( __traits( compiles, valloc ) )
-            {
-                free( m_pmem );
-            }
-            else static if ( __traits( compiles, malloc ) )
+            else
             {
                 free( m_pmem );
             }
         }
+        else
+            static assert(false);
```

Comment:
```suggestion
            static assert(false, "`Fiber` support is not available on this platform, please raise an issue");
```


Or something similar.

Reply:
Not needed for now - there are no easy ways to get to versions other than windows or posix.

Reply:
It's not rare to have people stepping in and trying to port D to an architecture / OS we currently do not support.
Also, some people do funky stuff with druntime, building their own.
Those errors message ensure that they get *some kind of clue* of what is failing at a minimum cost to us.

Reply:
If everything was as you say, then malloc already would work. :-)

Proposed text is a comment, and I consider what obvious comments like this are bad.

Reply:
On the contrary, the next poor schmuck that tries to build for a micro controller, e.g. this guy https://forum.dlang.org/thread/erwfgtigvcciohllvaos@forum.dlang.org , is going to hit that error and get 

```
src/core/thread/fiber.d(1018): Error: static assert:  *false* is false.
```
Rather than 
```
src/core/thread/fiber.d(1018): Error: static assert:  "*Fiber* support is not available on this platform, please raise an issue"
```

Reply:
She uses Posix, as I understand